nimble/host/include/host/ble_sm.h

```diff
@@ -132,8 +329,22 @@ int ble_sm_csis_resolve_rsi(const uint8_t *rsi, const uint8_t *sirk,
 #endif
 
 #if NIMBLE_BLE_SM
+/**
+ * @brief Passes the IO data from an application to the Security Manager during the pairing
+ * process.
+ *
+ * It should be used after a pairing method has been established for given connection
+ * and once the appropriate key has been generated.
```

Comment:
and once the appropriate key generation information (eg passkey) has been obtained.